stable/kube-state-metrics/values.yaml

```diff
@@ -114,8 +114,8 @@ collectors:
   services: true
   statefulsets: true
   storageclasses: true
-  validatingwebhookconfigurations: false
-  verticalpodautoscalers: false
+  validatingwebhookconfigurations: true
+  verticalpodautoscalers: true
   volumeattachments: false
```

Comment:
Let’s move volumeattschments to maintain lexical order and enable that as well.

Reply:
Not sure where you want to move it to? Order seems right to me. Good catch, I enabled it.

Reply:
Oh my bad. Nvm :)